test/parser-test.js

```diff
@@ -429,6 +429,7 @@ describe('parser#parseFilter()', function() {
 
 describe('parser#parse()', function() {
   it('should parse EasyList without failing', function(cb) {
+    this.timeout(5000);
```

Comment:
I think this was intermittent, completing just over the ~2000ms default timeout in some cases. The new babel is supposed to be faster, but maybe the mocha compiler is a bit slower causing this to slow down a tad? I suppose I should've timed it before/after this change.